libflux/flux-core/src/bin/fluxdoc.rs

```diff
@@ -174,13 +175,16 @@ fn lint(
     };
     let (_, mut diagnostics) = parse_docs(stdlib_dir, dir, exceptions)?;
     if !diagnostics.is_empty() {
-        let rest = diagnostics.len() as i64 - limit as i64;
+        let rest = match limit {
```

Comment:
Seems odd that this logic is needed. I expected rest to be a signed integer thus allowing it to become negative if the limit was greater than the actual length of the list, thus keeping the logic below for `if rest > 0` simple as well. Can we still explicitly use a signed integer for rest? Maybe add a comment that we are using a signed integer so that we rest can be negative?

Reply:
I understood it like this: if the limit is unlimited, then the remaining lints to print will always be 0!

I'd happy to change it, though. 😇